docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -699,7 +699,6 @@ You can see them also by typing ```%%cython?`` in IPython or a Jupyter notebook.
 
 ============================================  =======================================================================================================================================
 
-
```

Comment:
Accidental removal?
I consider it a good thing to keep double empty lines between sections, to give them a better visual separation.